prow/flagutil/kubernetes_cluster_clients.go

```diff
@@ -174,6 +174,23 @@ func (o *KubernetesOptions) InfrastructureClusterClient(dryRun bool) (kubernetes
 	return o.kubernetesClientsByContext[kube.InClusterContext], nil
 }
```

Comment:
`InfrastructureClusterClient` can now be simplified to `return o.ClusterClientForContext(kube.InClusterContext, dryRun)`